ci-operator/step-registry/baremetalds/e2e/test/baremetalds-e2e-test-commands.sh

```diff
@@ -183,6 +179,13 @@ do
 done
 echo "$(date) - all imagestreams are imported."
 
+# additional check to ensure that all the co didn't switch back meanwhile
+echo "$(date) - Waiting 10 minutes before checking again clusteroperators"
+sleep 10m
```

Comment:
The first wait may have taken 10 minutes (sometimes I think it works and does what we expect), maybe its better to add this sleep at the very start so we don't end up adding 20 minutes to the Job run time.

Reply:
Yeah, probably we could just add the sleep at the very beginning and keep just one `oc wait`. I'd like to see now the output of some sample runs with this approach, we'll push the change later

Reply:
I can see that there were some cases were 10 minutes timeout on the `oc wait` wasn't enough, so anticipating the sleep could also help for that